config/app/com/microsoft/azure/iotsolutions/uiconfig/webservice/v1/models/PackageListApiModel.java

```diff
@@ -43,4 +43,19 @@ public PackageListApiModel(Iterable<Package> models) {
         this.metadata.put("$type", String.format("Package;%s", Version.Number));
         this.metadata.put("$url", String.format("/%s/packages", Version.Path));
     }
+
+    public PackageListApiModel(Iterable<Package> models, String type, String config) {
+
+        this.items = StreamSupport.stream(models.spliterator(), false)
```

Comment:
can you add a comment as to what the parameters for this constructor mean/what the expected behavior is